apps/backend/app/api/routes/feature_store.py

```
@@ -286,6 +286,8 @@ async def update_feature(
             created_at=feature.created_at.isoformat(),
             updated_at=feature.updated_at.isoformat()
         )
+    except ValidationError as e:
+        raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_ENTITY, detail=str(e))
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)